components/eventing-controller/pkg/env/nats_config.go

```diff
@@ -9,7 +9,7 @@ import (
 // NATSConfig represents the environment config for the Eventing Controller with Nats.
 type NATSConfig struct {
 	// Following details are for eventing-controller to communicate to Nats
-	URL           string `envconfig:"NATS_URL" required:"true"`
```

Comment:
I had to make this change to to make it usable for Eventing Manager as it builds the nats url on the fly.